.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -160,6 +160,15 @@ jobs:
           TEST_DEVICE: CPU
           USE_SYSTEM_CACHE: False
 
+      - name: RoPE Test
```

Comment:
let us rename it because this job will check not only RoPE transformations

Reply:
I will rename it in the next PR

Reply:
we are waiting for this PR